packages/client/src/views/GrantDetailsView.vue

```diff
@@ -29,6 +29,65 @@
           {{ currentGrant.title }}
         </h2>
 
+        <!-- Left page column: main print/copy/grants.gov buttons -->
+        <div class="grant-details-main-actions print-d-none">
+          <div class="d-flex justify-content-between align-items-center">
+            <b-button
+              variant="primary"
+              block
+              class="mr-5"
+              style="width: fit-content; white-space: nowrap; "
+              :href="`https://www.grants.gov/search-results-detail/${currentGrant.grant_id}`"
+              target="_blank"
+              rel="noopener noreferrer"
+              data-dd-action-name="view on grants.gov"
+              @click="onOpenGrantsGov"
+            >
+              <b-icon
+                icon="box-arrow-up-right"
+                aria-hidden="true"
+                class="mr-2"
+              />
+              View on Grants.gov
+            </b-button>
+            <div class="w-20 d-flex">
+              <a
+                class="link-primary"
+                role="button"
+                :variant="copyUrlSuccessTimeout === null ? 'outline-primary' : 'outline-success'"
+                data-dd-action-name="copy btn"
+                style="white-space: nowrap; "
```

Comment:
In general, we should avoid inline styles where possible, since they're harder to maintain, keep consistent, override, etc. Instead, we want to favor existing [utility classes](https://bootstrap-vue.org/docs/reference/utility-classes) where available, or styles defined in a `style` section of the component otherwise. 

In this case, you should be able to use the `text-nowrap` utility class provided by Bootstrap. ([Docs here](https://getbootstrap.com/docs/4.5/utilities/text/#text-wrapping-and-overflow))

Reply:
thanks! getting used to Bootstrap

Reply:
Sigh... me too... :P